src/main/java/uk/gov/hmcts/reform/idam/web/AppController.java

```diff
@@ -414,6 +416,20 @@ private void getLoginFailureReason(HttpStatusCodeException hex, Model model, Bin
         }
     }
 
+    @GetMapping("/verification")
+    public String verificationView(
+        @ModelAttribute("verificationCommand") VerificationRequest request, BindingResult bindingResult, Model model) {
+        return VERIFICATION_VIEW;
+    }
+
+    @PostMapping("/verification")
+    public String verification(
+        @ModelAttribute("verificationCommand") VerificationRequest request, BindingResult bindingResult, Model model) {
+        //TODO push code to forgerock and verify: request.getCode();
```

Comment:
Are these TODO before this gets merged?

Reply:
These are TODOs to be finished in another ticket